armi/materials/material.py

```diff
@@ -845,9 +856,36 @@ class FuelMaterial(Material):
     class1_wt_frac = None
     class1_custom_isotopics = None
     class2_custom_isotopics = None
-    puFrac = 0.0
-    uFrac = 0.0
-    zrFrac = 0.0
+    puFracDefault = 0.0
+    uFracDefault = 0.0
+    zrFracDefault = 0.0
```

Comment:
we need to be clear whether this is weight fraction or atomic fraction. Prefer `puWtFracDefault`. The `puFrac` parameter in ARMI is not weight fraction of total mass, but instead weight fraction of heavy metal.